### DISCUSSION TITLE
just checking in on you!

### DISCUSSION THREAD

**Discussion by DeceptedPar** [MrMeowzz/Funkin-MrMeowzz#23] [General]
how are ya doin? what are you currently workin on for the mod?

**Comment by MrMeowzz** [comment]
nothing much, just haven't had time to work on it

**Reply by DeceptedPar** [reply]
ah, why is that?

**Reply by MrMeowzz** [reply]
school + I don't feel like working on it sometimes

**Reply by DeceptedPar** [reply]
eh it's alright, I'm not forcing to work on it, we all take breaks sometimes

**Reply by DeceptedPar** [reply]
maybe it's also cause of lack of motivation/ideas?

**Reply by MrMeowzz** [reply]
yeah i do need more ideas

**Reply by DeceptedPar** [reply]
maybe add more arrow typed, like halo notes, fire notes, caution sign notes, the poison notes from the bob mod, 5-10 keys

**Reply by DeceptedPar** [reply]
also update the debug menu by taking kades code

**Reply by DeceptedPar** [reply]
I can kinda help you about the arrow types - ideas
take the code from the shaggyxmatt debug menu, use that as a base for the new arrow typesthe gui you see on the left on the debug menu can also be used in this, it'll spruce thing up a bit

**Reply by DeceptedPar** [reply]
there's another fnf mod I'm helping aswell. I gave them this idea aswell, it works quite well aswell

**Reply by ilovevanessa11** [reply]
Hey MrMeowzz i was wondering if you could help me?

**Reply by MrMeowzz** [reply]
With what?

**Reply by ilovevanessa11** [reply]
making an fnf mod

**Reply by ilovevanessa11** [reply]
also please make a bside week 7 and please increase the speed on normal hard
